modules/SDL.lua

```diff
@@ -33,33 +33,29 @@ function SDL:StartSDL(pathToSDL, smartDeviceLinkCore, ExitOnCrash)
   end
   local status = self:CheckStatusSDL()
 
-  while status == self.RUNNING do
-    sleep(1)
-    print('Waiting for SDL shutdown')
-    status = self:CheckStatusSDL()
+  if (status == self.RUNNING) then
+    local msg = "SDL had already started out of ATF"
+    xmlReporter.AddMessage("StartSDL", {["message"] = msg})
+    print(console.setattr(msg, "cyan", 1))
+    return nil, msg
   end
 
-  if status == self.STOPPED  or status == self.CRASH then
-    CopyInterface()
-    local result = os.execute ('./tools/StartSDL.sh ' .. pathToSDL .. ' ' .. smartDeviceLinkCore)
-    if config.storeFullSDLLogs == true then
-      sdl_logger.init_log(get_script_file_name())
-    end
-    if result then
-      local msg = "SDL started"
-      xmlReporter.AddMessage("StartSDL", {["message"] = msg})
-      return true
-    else
-      local msg = "SDL had already started not from ATF or unexpectedly crashed"
-      xmlReporter.AddMessage("StartSDL", {["message"] = msg})
-      print(console.setattr(msg, "cyan", 1))
-      return nil, msg
-    end
+  CopyInterface()
+  local result = os.execute ('./tools/StartSDL.sh ' .. pathToSDL .. ' ' .. smartDeviceLinkCore)
```

Comment:
@LuxoftAKutsan StartSDL.sh uses 1st agrument as a directory, so it works correctly with and without '/'